pkg/pxc/service.go

```diff
@@ -407,8 +412,8 @@ func NewServiceHAProxyReplicas(cr *api.PerconaXtraDBCluster, owners ...metav1.Ow
 	return obj
 }
 
-func fillServiceLabels(labels map[string]string, pod *api.PodSpec) map[string]string {
-	for k, v := range pod.ServiceLabels {
+func fillServiceLabels(labels map[string]string, serviceLabels *map[string]string) map[string]string {
```

Comment:
Hey, is there a reason why you accept a pointer to a map? The map is already a reference type, so I guess you could have it just like `serviceLabels map[string]string`. With this, you don't have to dereference it on line 416. Also, you don't have to pass maps to it with a pointer operator.
Or am I missing something here?